discovery/snowflake/spcs.go

```diff
@@ -0,0 +1,440 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
```

Comment:
I don't think you can reserve all your rights while contributing to an Apache-licenced project.
However I am not a lawyer.  If necessary we can get the CNCF's lawyers to weigh in, but I would
ask as a simple alternative that you add the same banner found in all other Prometheus files.

Reply:
I will be adding the same banner found in other prometheus files. thanks for pointing.